packages/apps/haaretz.co.il/utils/getHomePageElements.js

```diff
@@ -73,6 +74,7 @@ const elements = new Map([
   [ 'Morbo', Morbo, ],
   [ 'Panucci', Panucci, ],
   [ 'Pazuzu', Pazuzu, ],
+  [ 'Roberto', Roberto, ],
```

Comment:
Roberto is really very rarely used, so lets only dynamically import it when needed.

My suggestion is that if the element in the map is a component, use the component, if it's a `string`, use it to dynamically import from `htz-components`. So here change to:
```suggestion
  [ 'Roberto', 'Roberto', ],
```
and then change `getHomepageElements` to:
```diff
export default function getHomepageElements(inputTemplate) {
+  const element = elements.get(inputTemplate) || DefaultComponent;
+  const Component = typeof element === 'string'
+    ? dynamic(() => import('@haaretz/htz-components').then(module => module[element]
+    : element;
+  return Component;
-  return elements.get(inputTemplate) || DefaultComponent;	  return elements.get(inputTemplate) || DefaultComponent;
}	}
```

I'm open to other suggestions on how to do this though